lessons/02-express-templating/README.md

```diff
@@ -370,7 +388,7 @@ module.exports = router;
 
 Carefully reading the code above should be enough to get you started. Note that handlebars templating is not used in the example above, but you are required to use handlebars templating for your assignment. So you should be using *res.render*, not *res.send*! And though it isn't nessecary to, you can modify the fakeDatabase.js code if you wish.
 
-#### Assignment 
+##Assignment 
```

Comment:
Somewhere in this section, we should be explicit that we're following our homework rubric for this assignment, and then maybe also acknowledge the javascript.md, airbnb style guide , Node beginner book and other things as useful resources that'll help them out.